gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -185,21 +184,8 @@ public GrpcCallContext nullToSelf(ApiCallContext inputContext) {
 
   @Override
   public GrpcCallContext withCredentials(Credentials newCredentials) {
-    Preconditions.checkNotNull(newCredentials);
-    CallCredentials callCredentials = MoreCallCredentials.from(newCredentials);
-    return new GrpcCallContext(
-        channel,
-        newCredentials,
-        callOptions.withCallCredentials(callCredentials),
-        timeout,
-        streamWaitTimeout,
-        streamIdleTimeout,
-        channelAffinity,
-        extraHeaders,
-        options,
-        retrySettings,
-        retryableCodes,
-        endpointContext);
+    // Credentials will be attached to the gRPC transport channels.
+    return this;
```

Comment:
I believe we still need the credentials value to help with validating the universe domain: https://github.com/googleapis/sdk-platform-java/blob/651cc9a7beabf8534d5b6edc83c1c554a9e691ef/gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java#L689-L711

Can this just no-op only for the callOptions portion?

Reply:
You are absolutely right. Sorry I was naive. Done.

Reply:
No worries at all. It's a completely separate feature.